hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/ResetDeletedBlockRetryCountSubcommand.java

```diff
@@ -41,25 +50,45 @@ public class ResetDeletedBlockRetryCountSubcommand extends ScmSubcommand {
 
   static class TransactionsOption {
     @CommandLine.Option(names = {"-a", "--all"},
-        description = "reset all expired deleted block transaction retry" +
+        description = "Reset all expired deleted block transaction retry" +
             " count from -1 to 0.")
     private boolean resetAll;
 
     @CommandLine.Option(names = {"-l", "--list"},
         split = ",",
-        description = "reset the only given deletedBlock transaction ID" +
+        paramLabel = "txId",
+        description = "Reset the only given deletedBlock transaction ID" +
             " list, e.g 100,101,102.(Separated by ',')")
     private List<Long> txList;
-  }
 
-  @CommandLine.ParentCommand
-  private ScmAdmin parent;
+    @CommandLine.Option(names = {"-i", "--in"},
+        description = "Use file as input, need to be JSON Array format and" +
+            " contains multi \"txID\" key.")
+    private String fileName;
+  }
 
   @Override
   public void execute(ScmClient client) throws IOException {
     int count;
     if (group.resetAll) {
       count = client.resetDeletedBlockRetryCount(new ArrayList<>());
+    } else if (group.fileName != null) {
+      Gson gson = new Gson();
+      List<Long> txIds;
+      try (InputStream in = new FileInputStream(group.fileName);
+           Reader fileReader = new InputStreamReader(in,
+               StandardCharsets.UTF_8)) {
+        DeletedBlocksTransactionInfoWrapper[] txns = gson.fromJson(fileReader,
+            DeletedBlocksTransactionInfoWrapper[].class);
+        txIds = Arrays.stream(txns).map(DeletedBlocksTransactionInfoWrapper::
+            getTxID).distinct().collect(Collectors.toList());
+        System.out.println("Load txIDS: " + txIds);
```

Comment:
Please output the start txID, and total number of txIds here.

Reply:
![image](https://user-images.githubusercontent.com/10106574/222427449-30a53002-aa60-4e67-a60c-21a540ef96e3.png)